accounts/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
I wonder if this file is even required here...
Maybe you should remove it, since it's (almost) empty and without any ToDo's?